phpunit.xml.dist

```diff
@@ -1,27 +1,26 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" colors="true" bootstrap="tests/bootstrap.php"
-         convertDeprecationsToExceptions="false"
          xsi:noNamespaceSchemaLocation="vendor/bin/.phpunit/phpunit/phpunit.xsd">
```

Comment:
On my project i am using phpunit v12, but can you check if these are available when running on v10?:
```xml
<phpunit
         displayDetailsOnIncompleteTests="true"
         displayDetailsOnPhpunitDeprecations="true"
         displayDetailsOnTestsThatTriggerDeprecations="true"
         displayDetailsOnTestsThatTriggerErrors="true"
         displayDetailsOnTestsThatTriggerNotices="true"
         displayDetailsOnTestsThatTriggerWarnings="true"
         failOnDeprecation="false"
         failOnNotice="true"
         failOnWarning="true"
         failOnRisky="true"
>
```

```xml
<source ignoreSuppressionOfDeprecations="true" ignoreIndirectDeprecations="true" restrictNotices="true" restrictWarnings="true">
```

Reply:
Well there are two `auto_mapping` options.
One is under `doctrine.orm.auto_mapping` path and second one under `doctrine.orm.controller_resolver.auto_mapping`. Maybe you disabled the first one? :)

Reply:
I did most of work about removing doctrine annotations in https://github.com/meilisearch/meilisearch-symfony/pull/424